RecipeApp.Core/RecipeConverter.cs

```diff
@@ -1,14 +1,176 @@
 ﻿using System;
 using System.Globalization;
+using System.Collections.Generic;
+using System.Linq;
 using System.IO;
 using System.Net;
 
 namespace RecipeApp
 {
     public class RecipeConverter
     {
-        static double[] multipliers = {240, 14.79, 4.93};
-        static string[] units = {"cup", "tablespoon", "teaspoon"};
+        static double[] multipliers = {240,3790,29.57,470,950,14.79,4.93};
+        static string[] units = {"cup","gallon","fluid ounce","pint","quart","tablespoon","teaspoon"};
+        static List<string> doubleUnits = new List<string>();
+
+        // Entry method to get some initial setup for Lists
+        public static void InitialSetup() {
+            PopulateDoubleUnitsList();
+            SortCookingUnitsByMultiplier();
+        }
+
+        public static string RecipeValidation(string recipeText) {
```

Comment:
Methods do things, therefore they should be named after verbs rather than nouns. rename to: `ValidateRecipe`